src/main/java/de/tum/in/www1/artemis/web/rest/dto/LectureUnitInformationDTO.java

```diff
@@ -2,5 +2,6 @@
 
 import java.util.List;
 
-public record LectureUnitInformationDTO(List<LectureUnitSplitDTO> units, int numberOfPages, boolean removeBreakSlides, boolean removeSolutionSlides) {
+public record LectureUnitInformationDTO(List<LectureUnitSplitDTO> units, int numberOfPages, boolean removeBreakSlides, String removeBreakSlidesKeyword,
```

Comment:
```suggestion
@JsonInclude(Include. NON_NULL)
public record LectureUnitInformationDTO(List<LectureUnitSplitDTO> units, int numberOfPages, boolean removeBreakSlides, String removeBreakSlidesKeyword,
```
This is the annotation I was referring to.

Reply:
See https://github.com/ls1intum/Artemis/pull/7060#discussion_r1294725354